deploy/.env

```diff
@@ -0,0 +1,18 @@
+BACKOFFICE_PORT=5137
+HEADLESS_SVC_PORT=5173
+WORKFLOW_SVC_PORT=3000
+BCRYPT_SALT=10
+JWT_SECRET_KEY=secret
+JWT_EXPIRATION="10d"
+API_KEY="secret"
+NODE_ENV="development"
+COMPOSE_PROJECT_NAME=ballerine-x
+DB_PORT=5432
+DB_USER=admin
+DB_PASSWORD=admin
+SESSION_SECRET=iGdnj4A0YOhj8dHJK7IWSvQKEZsG7P70FFehuddhFPjtg/bSkzFejYILk4Xue6Ilx9y3IAwzR8pV1gb4
```

Comment:
Do not keep secrets in the code (even if they are mock secrets - just keep them blank, the reason is we can know what is mock and what is not)

Reply:
or just have a string "secret" instead